samples/apply-hillshade-renderer-to-raster/README.md

```diff
@@ -0,0 +1,32 @@
+# Apply hillshade renderer to raster
+
+Apply a hillshade renderer to a raster.
+
+![Image of apply hillshade renderer to raster](apply-hillshade-renderer-to-raster.png)
+
+## Use case
+
+An environmental agency may track coastal erosion by comparing images of an area taken over a a longer period of time with hillshade renderers applied.
+
+## How to use the sample
+
+Choose and adjust the settings to update the hillshade renderer on the raster layer. The sample allows you to change the Altitude, Azimuth, and Slope Type.
+
+## How it works
+
+1. Create a `Raster` from a grayscale raster file.
+2. Create a `RasterLayer` from the raster.
+3. Create a `Basemap` from the raster layer and set it to the map.
```

Comment:
this is not being done in the ViewModel. Instead you're adding the raster layer to the arcGISMap operational layers